flixel/FlxGame.hx

```diff
@@ -243,6 +243,14 @@ class FlxGame extends Sprite
 		// Super high priority init stuff
 		_inputContainer = new Sprite();
 		
+		if (GameSizeX <= 0)
```

Comment:
I meant windowWidth x windowHeight.

What I meant is that it seems confusing to the reader that random negative values would turn into a `FlxGame` instance which uses the window dimensions, wheareas with `0`, you can at least guess that it probably has some special meaning.